src/components/buttonSelectors/MultiButtonSelector.js

```diff
@@ -30,7 +32,20 @@ const MultiButtonSelector = (props) => {
 
 
     const submitField = (optionText, idCode) => {
-        setSelection(optionText);
+//        setSelection(optionText);
+        let selectionValidation = SelectionValidation
```

Comment:
is there a reason that you set it to a local variable and don't just call it directly?